src/web/builds.rs

```diff
@@ -27,6 +57,58 @@ fn serde_relative_time<S: Serializer>(
     time::at(*ts).rfc3339().to_string().serialize(serializer)
 }
 
+/// The list of builds
```

Comment:
```suggestion
/// All builds for a specific release of a crate.
///
/// In most cases, there will only be one build.
/// A release may have multiple builds if a server administrator decides to requeue a build.
/// Similarly, it may have no builds if the crate was removed from the queue before being built.
```